tests/unit/server/bundle_store_test.py

```diff
@@ -3,6 +3,41 @@
 
 
 class BundleStoreTest(BaseBundleManagerTest):
+    def test_add_bundle_store(self):
```

Comment:
This test tests a bunch of endpoints, but you called it "test_add_bundle_store" and said "Tests the workflow for creating bundles" in the docstring. Can you update both so it better reflects what this test does?